src/posix/platform/radio.cpp

```diff
@@ -205,6 +209,56 @@ void Radio::Init(void)
 
 void *Radio::GetSpinelInstance(void) { return &sRadioSpinel; }
 
+void Radio::GetIidListFromRadioUrl(spinel_iid_t *aIidList, const uint8_t aLength)
+{
+    // Validate the arguments
```

Comment:
Remove the simple comments?

I used to to add similar comments (and sometimes still do) but try to edit myself. :) 
My approach is if the comment is just explaining the statements that follows without adding any new insight/info we don't need it and can remove it.

```suggestion
```